Gemfile

```diff
@@ -16,9 +15,7 @@ group :jekyll_plugins do
   gem 'jekyll_frontmatter_tests'
 end
 
-
-gem 'hash-joiner'
-gem 'safe_yaml'
```

Comment:
Someone can't tell if these gems were needed or not from a glance, but seeing as [the site builds](https://travis-ci.org/18F/18f.gsa.gov/builds/91811785), doesn't seem that they are missed!